search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -35,6 +36,7 @@ function onClickRegion(e) {
   displayTopResultsForCurrentTrend(countryId);
   setCountryTrends(countryId);
   displaySentimentChartForCurrentTrend(countryId);
+  displayPopularityTimeline(countryId);
```

Comment:
Can you rename all these functions to be `set...` like `setPopularityTimeline` bc here we are only setting the content under the tab but they are only displayed when the tab is clicked.

Also, consider rearranging the functions according to the order of the tabs (both function calls here and function bodies).

Reply:
In the resize all charts, the pr that's going to be sent out right after this one, the charts methods only called when they are resized so in that sense it is displaying, and it is rearragned correctly in order in that branch. We can discuss further in that PR.